file_paths.md

```diff
@@ -275,6 +279,12 @@ create a `puppet` user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        bin *
+            pxp-agent.exe *
+        pxp-modules *
+            pxp-module-puppet *
```

Comment:
I'm differentiating the directory name from the Puppet "modules" one.

This is just a proposal; no strong opinion.